cloud-provider-aws.yaml

```diff
@@ -0,0 +1,67 @@
+package:
+  name: cloud-provider-aws
+  version: 1.31.1
+  epoch: 0
+  description: The AWS cloud provider provides the interface between a Kubernetes cluster and AWS service APIs.
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  environment:
+    CGO_ENABLED: "0"
+    GO111MODULE: "on"
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/kubernetes/cloud-provider-aws
+      tag: v${{package.version}}
+      expected-commit: 94cb7eefd72f6577a0f7840f61a47ebd4aed06f4
+
+subpackages:
+  - name: aws-cloud-controller-manager
+    description: The AWS Cloud Controller Manager is the controller that is primarily responsible for creating and updating AWS loadbalancers (classic and NLB) and node lifecycle management.
+    pipeline:
+      - uses: go/build
+        with:
+          packages: ./cmd/aws-cloud-controller-manager
+          output: aws-cloud-controller-manager
+          ldflags: |
+            -w -s
+            -X k8s.io/component-base/version.gitVersion=v${{package.version}}
+            -X main.gitVersion=v${{package.version}}
+    test:
+      pipeline:
+        - runs: |
+            aws-cloud-controller-manager --help
+            aws-cloud-controller-manager --version
+
+  - name: ecr-credential-provider
```

Comment:
ecr-credential-provider is not part the image so I think we can omit that part from the package. (this is not a blocker, we can package both inside here as well and use `${{package.name}}-foo`

Reply:
I added it `ecr-credential-provider`, in case it was needed in the future.